pages/getting-access.md

```diff
@@ -79,3 +79,51 @@ allocation.
   request a CCR account and VPN access by [contacting CCR Help](help.md)
 
 - __Industry customers__ please [contact CCR Help](help.md) to request access.  
+
+## Available Resources  
```

Comment:
This all looks great. What about making this it's own page though? I think having it as a first class menu option on the left side would be great?

Reply:
Moved to separate page and added some additional info